doc/source/qpoint-phonon-modes.rst

```diff
@@ -71,14 +71,29 @@ Calculating The Coherent Neutron Structure Factor
 
 The neutron structure factor can be calculated for each branch and q-point
 using :py:meth:`QpointPhononModes.calculate_structure_factor <euphonic.qpoint_phonon_modes.QpointPhononModes.calculate_structure_factor>`
```

Comment:
Add 'From this function a :ref:`StructureFactor<structure-factor>` object is returned' here. Or maybe that's just superfluous and a user could figure that out from the docstring anyway...